packages/terra-icon/tests/wdio/icon-spec.js

```diff
@@ -0,0 +1,38 @@
+/* global browser, Terra, before */
+describe('Icon', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-default'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Append', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-append-class'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Aria', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-aria'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('HeightWidth', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-height-width'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Bidi', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-bidi'));
```

Comment:
Need to add `dir="rtl"` to the [div here](https://github.com/cerner/terra-core/blob/master/packages/terra-icon/tests/nightwatch/components/IconBidi.jsx#L7) so the `isBidi` prop behavior is visible in the screenshot.

Reply:
Brett did this in a commit of his own.